### PR TITLE
feat: add sort gridEvent(#515)

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

#515 

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/dispatch/data.ts [hunk 1]
```diff
@@ -192,6 +192,19 @@ export function sort(store: Store, columnName: string, ascending: boolean) {
...
   }
+
+  const eventBus = getEventBus(store.id);
+  const gridEvent = new GridEvent({ sortOptions });
+
+  /**
+   * Occurs when sort.
+   * @event Grid#sort
+   * @property {Event} nativeEvent - Event object
+   * @property {string} targetType - Type of event target
```
Comment (js87zz) [inline src/dispatch/data.ts]: `nativeEvent`, `targetType`는 주석에서 제거되어야 될 것 같습니다!

File: src/dispatch/data.ts [hunk 2]
```diff
@@ -192,6 +192,19 @@ export function sort(store: Store, columnName: string, ascending: boolean) {
     data.rawData = rawData;
     data.viewData = viewData;
   }
+
+  const eventBus = getEventBus(store.id);
+  const gridEvent = new GridEvent({ sortOptions });
```
Comment (js87zz) [inline src/dispatch/data.ts]: 기존 sortOptions는 변경되기 전 상태이기 때문에 `new GridEvent({ sortOptions: data.sortOptions })`로 변경해야될 것 같습니다.

File: src/dispatch/data.ts [hunk 3]
```diff
@@ -192,6 +192,19 @@ export function sort(store: Store, columnName: string, ascending: boolean) {
...
+
+  /**
+   * Occurs when sort.
```
Comment (seonim-ryu) [inline src/dispatch/data.ts]: sort -> sorting 이요ㅎㅎ

File: src/dispatch/data.ts (hunk 2 above)
Comment (jung-han) [reply src/dispatch/data.ts]: 헛 수정하였습니다.

Comment (seonim-ryu) [reply src/dispatch/data.ts]: 명시적으로 `changeSortBtn`에서 값을 반환하고 그 값을 이용하는건 어떨까요??
`changeSortBtn` 호출로 인해서 `data.sortOptions` 값이 변경된게 잘 인지되지 않는것 같아요 ;ㅅ;
```js
function changeSortBtn() {
  // ...
  return data.sortOptions;
}

const chagnedSortOptions = changeSortBtn( /**/ );
// ...
const gridEvent = new GridEvent({ sortOptions: chagnedSortOptions });
```

**Comment by seonim-ryu** [comment]
[7/16] 리뷰 완료! 고생하셨습니다ㅎ

**Comment by jung-han** [comment]
ok to test

**Comment by jung-han** [comment]
ok to test

**Pull request merged**